bcgov/biohubbc
# Hot Fix: Added cases for `ensureProtocol` method; Fix project participant editing

# Overview
 - Fixes a bug with editing project participants
 - Modifies the `ensureProtocol` utiltiy function to match the BioHub implementation

## Testing Procedure
1. Attempt to edit a project's participants in dev. It should fail.
2. `git checkout ensure-protocol`
3. You should now be able to edit project participants.
